prow/jobs/kyma/skr-svcat-migration.yaml

```diff
@@ -0,0 +1,55 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+periodics: # runs on schedule
+    - name: skr-azure-svcat-migration-dev
+      annotations:
+        pipeline.clusterprovisioning: "keb"
+        pipeline.installer: "keb"
+        pipeline.platform: "gardener_azure"
+        pipeline.test: "fast-integration"
+        pipeline.trigger: "periodic"
+        testgrid-create-test-group: "false"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "skr-azure-svcat-migration-dev"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-btp-operator-stage-credentials: "true"
+        preset-build-main: "true"
+        preset-keb-dev-credentials: "true"
+      cron: "* */4 * * *"
+      skip_report: false
+      decorate: true
+      path_alias: github.com/kyma-project/kyma
+      cluster: trusted-workload
+      extra_refs:
+        - org: kyma-project
+          repo: kyma
+          path_alias: github.com/kyma-project/kyma
+          base_ref: main
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
+      spec:
+        containers:
+          - image: "wozniakjan/kyma-integration:123-k8s1.16"
```

Comment:
this is only for debugging purposes until https://github.com/kyma-project/test-infra/pull/3932 merges